jenkins/parser/paser-config-unittest.py

```diff
@@ -0,0 +1,50 @@
+import json
+import os
+
+jobs_config_path = "jobs-config.json"
+error_types = []
+
+with open(jobs_config_path, "r") as jobs_file:
+    jobs_object = json.load(jobs_file)
+    jenkins_jobs = jobs_object["jobsConfig"]["jenkinsJobs"]
+    error_msg = jobs_object["jobsConfig"]["errorMsg"]
+
+
+print("[TEST 1]: Checking that all jobs defined in jobs-config.json exist in Jenkins ...")
+job_names = [jenkins_jobs[job_id]["jobName"] for job_id in range(len(jenkins_jobs))]
+
+for job in job_names:
+    exit_code = os.system(
+        "ssh -o IdentitiesOnly=yes -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i /var/lib/jenkins/.ssh/id_rsa-openstack -l localcli -p 8090 localhost get-job "
+        + job
+        + " > /dev/null"
+    )
+    assert exit_code == 0, "Job " + job + " does not exists in Jenkins"
```

Comment:
@aandvalenzuela , I do not think this check is going to work as github is not going to run this on jenkins server node.  I think you can download https://raw.githubusercontent.com/cms-sw/cmssdt-wiki/master/jenkins_reports/All.md and then search for job names in it